doc/classes/RefCounted.xml

```diff
@@ -37,7 +37,7 @@
 			<return type="bool" />
 			<description>
 				Decrements the internal reference counter. Use this only if you really know what you are doing.
-				Returns [code]true[/code] if the decrement was successful, [code]false[/code] otherwise.
+				Returns [code]true[/code] if the decrement should free the object, [code]false[/code] otherwise.
```

Comment:
The current wording is true, but I'd adjust it a bit to make it 100% precise. I'm sorry, but I need to be as picky here as this is a dark corner with great implications.

```suggestion
				Returns [code]true[/code] if the object should be freed after the decrement, [code]false[/code] otherwise.
```